cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -48,6 +50,15 @@ enum {
 };
 
 /**
+ * @brief   Available peripheral buses
+ */
+enum {
+    AHB1,           /**< AHB1 bus */
+    AHB2,           /**< AHB2 bus */
+    AHB3            /**< AHB3 bus */
+};
```

Comment:
You highlighted a good point. Actually, the number of AHB is not the same for all stm32 CPUs. This 'enum' should be part of the specific periph_cpu.h.